elasticdl/python/worker/worker.py

```diff
@@ -21,7 +21,10 @@
     scatter_embedding_vector,
     string_to_id,
 )
-from elasticdl.python.common.log_utils import get_logger
+from elasticdl.python.common.log_utils import (
+    get_logger,
+    LOG_FILE,
```

Comment:
You can just use `LOG_FILE` inside `log_utils` and provide it as the default value for argument `filename`.